.github/CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# GitHub CODEOWNERS https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+lib/publiccloud/ @pdostal @grisu48
```

Comment:
And what about the others? I would add at least also @asmorodskyi and anyone else who wants to.

Reply:
It makes sense but I would like to wait for Anton.

Reply:
LGTM